src/providers/bitcoin/BitcoinLedgerProvider.js

```diff
@@ -161,6 +161,40 @@ export default class BitcoinLedgerProvider extends LedgerProvider {
     }))
   }
 
+  async getWalletInfo (numAddressPerCall = 10, from = {}) {
+    let addressIndex = from.index || 0
+    let unusedAddress = false
+    let usedAddresses = []
+    let balance = 0
+
+    while (!unusedAddress) {
+      let addresses = await this.getAddresses(addressIndex, numAddressPerCall)
+      const addressList = addresses.map(addr => addr.address)
+
+      const addressDeltas = await this.getMethod('getAddressDeltas')(addressList)
```

Comment:
@mattBlackDesign note that `getAddressDeltas` is a bitcore only function. With what we have before this, it will work both if you have a bitcoind or bitcore node because getUnspentTransactions is implemented in `BitcoinRPCProvider` and `BitcoreRPCProvider` but i don't see the same for `getAddressDeltas`.